app/docker/filters/filters.js

```diff
@@ -322,4 +322,8 @@ angular
       }
       return fullName.substring(0, versionIdx);
     };
+  })
+  .filter('containerportfilter', function () {
+    'use strict';
```

Comment:
"use strict" is not needed, as webpack enforces all our code to be strict

Reply:
> "use strict" is not needed, as webpack enforces all our code to be strict

The old code uses `strict`.

Reply:
yes, because it's old :) it was never refactored.

New code doesn't need it